src/api/spec/lib/rubocop/cop/view_component/avoid_global_state_spec.rb

```diff
@@ -26,16 +26,4 @@ def initialize
       RUBY
     end
   end
-
-  context 'when a class which is not a view component uses params' do
```

Comment:
This example needs to be removed since it doesn't apply anymore. It's not code restricting on which file the cop runs, but the new RuboCop configuration with `Include` defined in this PR.